wallet/how-to/use-non-evm-networks/starknet/manage-starknet-networks.md

```diff
@@ -0,0 +1,140 @@
+---
+description: Manage Starknet networks in MetaMask.
+sidebar_position: 3
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Manage Starknet networks
+
+You can detect a user's Starknet network and prompt them to switch Starknet networks in MetaMask,
+using the
+[`get-starknet`](https://github.com/starknet-io/get-starknet) library or the
+[`wallet_invokeSnap`](/snaps/reference/wallet-api-for-snaps/#wallet_invokesnap) JSON-RPC method.
+
+## Prerequisites
+
+[Connect to Starknet](connect-to-starknet.md) from your dapp.
+
+## Detect a user's network
+
+Detect the Starknet network a user is currently connected to using the following:
+
+<Tabs>
+  <TabItem value="get-starknet" default>
+
+  ```javascript
+  const checkCurrentNetwork = () => {
+    const starknet = getStarknet();
+    const currentNetwork = starknet.provider.rpcUrl;
+    console.log("Currently connected to:", currentNetwork);
+    return currentNetwork;
+  };
```

Comment:
```suggestion
  const checkCurrentNetwork = (wallet) => {
      try {
        if(wallet?.isConnected !== true){
          throw("Wallet not connected");
        } 
        const currentNetwork = wallet?.chchainId
        console.log("Currently connected to:", currentNetwork);
        return currentNetwork;
      } catch (error) {
        console.error("Error of detect current connected network:", error);
      }
  };
```

we should return the chainId , rather than rpcUrl, and standardized the example method as other